pyUltroid/_misc/_supporter.py

```diff
@@ -118,6 +118,7 @@ class Config((object)):
         PM_DATA = os.environ.get("PM_DATA", "ENABLE")
         DEEP_AI = os.environ.get("DEEP_AI", None)
         TAG_LOG = os.environ.get("TAG_LOG", None)
+        UFOPAPI = os.environ.get("UFOPAPI", "h32dfKsl8pQUxsNftkogIuDF32pYTwKluIY8emI1Hs")
```

Comment:
default API key would mean UFoP bans would be enabled by default for everyone. That wouldn't be required.

Reply:
Bans are not enabled unless it's truned on the same way Ultroidbans is turned on. The default key is so if anyone wants to they don't have to hunt down a key.

This key also works with Dalle-3-XL and OpenDalle I can add